### PR TITLE
Aleksei/apollo error handling

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

Error handling and preventing of triggering of the update function

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[0204/164418.758:ERROR:prune_crash_reports.cc(35)] PruneCrashReportDatabase: Failed to get pending reports
```
Comment (jbibla) [inline debug.log]: feels like this should be in the gitignore?

File: src/gql/apollo.js [hunk 2]
```diff
@@ -35,17 +37,50 @@ const makeWebSocketLink = () => {
...
+        sub = forward(operation).subscribe({
+          next: result => {
+            // check if we have errors
+            if (!result.errors) {
+              observer.next(result)
+            } else {
+              // pass errors to sentry
+              result.errors.map(err => {
+                Sentry.captureException(err, {
+                  level: "error"
```
Comment (faboweb) [inline src/gql/apollo.js]: why do we need to set the level?

Comment (iambeone) [reply src/gql/apollo.js]: Don't know for sure, just used as I have found in the example

Comment (faboweb) [reply src/gql/apollo.js]: we don't use this anywhere else, does it provide any additional value? if not let's remove it to stay coherent in the codebase.

Comment (iambeone) [reply src/gql/apollo.js]: Updated

**Pull request merged**